ts/Core/Chart/Chart.ts

```diff
@@ -2405,20 +2403,80 @@ class Chart {
         const tempWidth = chart.plotWidth;
 
         axes.some(function (axis: Axis): (boolean|undefined) {
+            // Left side must be align: right and right side must
+            // have align: left for labels
+            axis.reserveSpaceDefault = (
+                axis.side === 0 ||
+                axis.side === 2 ||
+                { 1: 'left', 3: 'right' }[axis.side] === axis.labelAlign
+            );
```

Comment:
I'm not sure if this was answered, but do we need repeating this code block? Since the whole logic of expected space only applies to horizontal axes (`axis.side` is always 0 or 2), this condition will always resolve to `true`.

Reply:
I felt, that I had forgotten about something... I agree that's true and shouldn't be here.